src/style.css

```
@@ -1 +1,45 @@
 /* add styles here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center center;
+  margin-right: 4px;
+}
+
+.stars--1 > .stars__star:nth-child(-n + 1) {
```

```suggestion
.stars--1 > .stars__star:nth-child(-n + 1),
.stars--2 > .stars__star:nth-child(-n + 2),
... {
```
Consider using commas to group your selectors